README.rst

```diff
@@ -39,8 +39,8 @@ flexible as possible.
     :target: https://github.com/astral-sh/ruff
     :alt: Code style
 
-.. |coverage| image:: https://img.shields.io/coverallsCoverage/github/vacanza/python-holidays?branch=main&color=41B5BE&style=flat
-    :target: https://coveralls.io/r/vacanza/python-holidays
+.. |coverage| image:: https://img.shields.io/codecov/c/github/vacanza/python-holidays/dev?color=41B5BE&style=flat&token=OnENZU5Gy4
```

Comment:
Without `token`, it also works. As I understand, token is needed for private repository coverage.

Reply:
Yeah, just checked that. Not sure if it was the case when I initially tested it.

Removed the unnecessary part.